Sources/SKCore/FallbackBuildSystem.swift

```diff
@@ -53,10 +53,10 @@ public final class FallbackBuildSystem: BuildSystem {
     if let sdkpath = sdkpath {
       args += [
         "-sdk",
-        sdkpath.asString,
+        sdkpath.description,
```

Comment:
`description` is for producing human readable path, please use `pathString` where you want to guarantee proper string representation.

Reply:
Thanks, will do.  I was wondering about this when I did it, and I chose `description` because that's what the SwiftPM change did. I didn't realize that `pathString` didn't exist at the time of that change.  It all makes more sense now.

Reply:
https://github.com/apple/sourcekit-lsp/pull/81